scene/resources/tile_set.cpp

```diff
@@ -5390,6 +5382,13 @@ Variant TileData::get_custom_data_by_layer_id(int p_layer_id) const {
 }
 
 bool TileData::_set(const StringName &p_name, const Variant &p_value) {
+#ifndef DISABLE_DEPRECATED
+	if (p_name == "texture_offset") {
+		texture_origin = p_value;
+		return true;
+	}
+#endif
```

Comment:
Usually we also handle `_get`. Not sure if it's strictly needed but that's what we normally do :P

Reply:
Hmm, that was not done for TileSet before (like, even in 3.x code). So I assumed it was only done in getters everywhere.

But well, I added it anyway.

Reply:
`_get` is only needed for script compatibility. If you handle only `_set`, at least your scene/resource will load without losing data (like in #71347).

Reply:
> `_get` is only needed for script compatibility. If you handle only `_set`, at least your scene/resource will load without losing data (like in #71347).

Yeah that's what I was thinking too. I am not sure a lot of people would set `texture_offset` without using the property (thus using `set(...)`)but I guess that's fine anyway.